docs/clients/rust/transactions.rst

```diff
@@ -0,0 +1,74 @@
+.. _ref_rust_transactions:
+
+Transactions
+------------
+
+The client also has a ``.transaction()`` method that
+allows for atomic `transactions`_.
+
+Wikipedia has a good example of a scenario requiring a transaction which we
+can then implement:
+
+.. pull-quote::
+
+  An example of an atomic transaction is a monetary transfer from bank account
+  A to account B. It consists of two operations, withdrawing the money from 
+  account A and saving it to account B. Performing these operations in
+  an atomic transaction ensures that the database remains in a consistent
+  state, that is, money is neither lost nor created if either of those two
+  operations fails.
```

Comment:
I like your solution here, but it doesn't seem to produce an actual HTML blockquote. I've tried some other methods for achieving this from various RST references, and none of those seem to work in our system either. I think the best way working within what we have is maybe to just italicize this paragraph.

```suggestion
*An example of an atomic transaction is a monetary transfer from bank account A
to account B. It consists of two operations, withdrawing the money from account
A and saving it to account B. Performing these operations in an atomic
transaction ensures that the database remains in a consistent state, that is,
money is neither lost nor created if either of those two operations fails.*
```